openedx/features/course_duration_limits/admin.py

```diff
@@ -3,16 +3,19 @@
 Django Admin pages for CourseDurationLimitConfig.
 """
 
-from __future__ import unicode_literals
+from __future__ import absolute_import, unicode_literals
 
 from django.contrib import admin
 from django.utils.translation import ugettext_lazy as _
 
 from openedx.core.djangoapps.config_model_utils.admin import StackedConfigModelAdmin
+
 from .models import CourseDurationLimitConfig
 
 
 class CourseDurationLimitConfigAdmin(StackedConfigModelAdmin):
+    """Admin for course duration limit
```

Comment:
nit: could you move this down a line? Like:
```
"""
Admin for course duration limit
"""
```

Reply:
Yes, I've moved it. Thanks!